charts/tidb-lightning/values.yaml

```diff
@@ -30,6 +30,8 @@ dataSource:
     storage: 100Gi
     secretName: cloud-storage-secret
     path: s3:bench-data-us/sysbench/sbtest_16_1e7.tar.gz
+    # Directory support downloading all files in a remote directory, shadow dataSoure.remote.path if present
```

Comment:
What about reusing the `path` for this? If a path ends with `/`, then it is a directory.

Reply:
I agree with the current solution, reusing the `path` might be more complex for developers and users.